src/main/resources/add-ons-to-index.json

```diff
@@ -894,6 +921,9 @@
       "uid": "org.openmrs.module.exportccd",
       "type": "OMOD",
       "name": "Export CCD Module",
+      "tags": [
+        "export", "ccd"
```

Comment:
It's not helpful to add a tag that only is used for one addon, and matches a word in the module's name. So remove "ccd". [Edit: later on I see there's another CCD, so it's okay to leave this]
Personally I don't think that "export" or "import" is very helpful if they are just repeating something that's in the title, so I would remove that (and I'll repeat this comment).